source/index.html

```diff
@@ -1,12 +1,90 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
-
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+<body>
+<header>
+  <div>
+    <a href="index.html"><img src="#" alt="Логотип Кэт Энерджи"></a>
+    <a href="index.html"><img src="#" alt="Текстовый логотип Кэт Энерджи"></a>
+    <button type="button" aria-label="Закрыть"></button>
+  </div>
+  <nav>
+    <ul>
+      <li><a href="index.html">Главная</a></li>
+      <li><a href="catalog.html">Каталог продукции</a></li>
+      <li><a href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main>
+  <h1>Функциональное питание для котов</h1>
+  <section>
+    <h2>Реклама питания для котов</h2>
+    <p>Занялся собой? Займись котом!</p>
+    <img src="#" alt="Коробка с питанием для котов и кошек Cat Energy Pro">
+    <a href="form.html">Подобрать программу</a>
+  </section>
+  <section>
+    <h2>Похудение</h2>
+    <p>Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+    <a href="#">Каталог Slim</a>
+  </section>
+  <section>
+    <h2>Набор массы</h2>
+    <p>Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+    <a href="#">Каталог pro</a>
+  </section>
+  <section>
+    <h2>Как это работает</h2>
+    <ul>
+      <li>Функциональное питание содержит только полезные питательные вещества.</li>
+      <li>Выпускается в виде порошка, который нужно лишь залить кипятком и готово.</li>
+      <li>Замените один-два приема обычной еды на наше функциональное питание.</li>
+      <li>Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</li>
+    </ul>
+  </section>
+  <section>
+<!--    Возможно вместо <section> лучше использовать <article>.-->
+    <h2>Живой пример</h2>
+    <p>Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+    <div>
+      <p>5 кг<span>снижение веса</span></p>
```

Comment:
```suggestion
      <p><span>5 кг</span><span>снижение веса</span></p>
```
Тоже возьми в спан 5 кг и 60 дней, чтобы во втором спане не переопределять потом стили от параграфа

Reply:
добавил спан